src/DynamicNLPModels.jl

```diff
@@ -312,8 +314,12 @@ Resulting `H`, `J`, `h`, and `h0` matrices are stored within `QuadraticModels.QP
 If `K` is defined, then `u` variables are replaced by `v` variables. The bounds on `u` are transformed into algebraic constraints,
 and `u` can be queried by `get_u` and `get_s` within `DynamicNLPModels.jl`
 """
-function DenseLQDynamicModel(dnlp::LQDynamicData{T,V,M}) where {T, V <: AbstractVector{T}, M  <: AbstractMatrix{T}, MK <: Union{Nothing, AbstractMatrix{T}}}
-    _build_dense_lq_dynamic_model(dnlp)
+function DenseLQDynamicModel(dnlp::LQDynamicData{T,V,M}; truncated = false) where {T, V <: AbstractVector{T}, M  <: AbstractMatrix{T}, MK <: Union{Nothing, AbstractMatrix{T}}}
```

Comment:
I would suggest documenting in the docstring the new optional argument `truncated`